openslides_backend/action/mixins/import_mixins.py

```diff
@@ -156,7 +157,9 @@ def add_item(self, entry: Dict[str, Any]) -> None:
             self.name_to_ids[key].append(entry)
 
 
-class BaseImportJsonUpload(SingularActionMixin):
+class BaseImportJsonUploadAction(SingularActionMixin, Action):
```

Comment:
Yes, but `SingularActionMixin` is supposed to be a mixin, as the name already suggests. At least in theory, a mixin should _mixed into_ the inheritance structure instead of being the sole ancestor. That `SingularActionMixin` also inherits from `Action` is more of a coincidence which should be ignored than a feature which should be used. But, yes, functionally, it does not make a difference in this case and I'm okay with reverting it if you're strongly against it.